shell/browser/api/electron_api_session.cc

```diff
@@ -182,6 +183,45 @@ struct Converter<ClearStorageDataOptions> {
   }
 };
 
+template <>
+struct Converter<electron::api::ExtensionTabDetails> {
+  static bool FromV8(v8::Isolate* isolate,
+                     v8::Local<v8::Value> val,
+                     electron::api::ExtensionTabDetails* out) {
+    gin_helper::Dictionary options;
+    if (!ConvertFromV8(isolate, val, &options))
+      return false;
+
+    *out = electron::api::ExtensionTabDetails();
+    auto tmp = electron::api::ExtensionTabDetails();
+
+    if (options.Get("windowId", &tmp.window_id))
```

Comment:
I think we can just do `options.Get("windowId", &out->window_id);` without using `tmp`?